### PR TITLE
added dockerfile

### PR DESCRIPTION
**Pull request by leisefuxX**
added and testet dockerfile

### CODE REVIEW

**Comment by leisefuxX** [comment]
didn't touch any code at all.
just commented favicon, css and html out in gosh.yml

File: dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# build stage
+## import image
+FROM golang:alpine AS build_base
+
+## add git
+RUN apk add --no-cache git
```
Comment (riotbib) [inline dockerfile:6]: I am uncertain why Git is needed in the Dockerfile. Could you explain your motive?

**Comment by riotbib** [comment]
Hej @leisefuxX thanks for your PR!

I guess @oxzi will want the Dockerfile to be moved under `contrib/docker/` e.g, as it's already done with the OpenBSD and NixOS deployment.

**Comment by oxzi** [comment]
Thanks @riotbib for your first review!

For the record, this PR was re-opened as #55.

**Pull request closed without merging**